GitHub Actions/haal-centraal-npm-beheer.md

```diff
@@ -160,6 +160,11 @@ Wil je met één statement alle packages updaten naar de meest recente versie, d
 - `ncu -u`. Met deze statement worden de npm package referenties in een package.json bestand ge-update naar de meest recente versie
 - `npm install`. Met deze statement worden de packages in de node_modules bestand geupdate aan de hand van het package.json bestand
 
+## Commit naar upstream repository
+
+Tijdens het hierboven beschreven proces wordt de folder 'node_modules' lokaal gegenereerd. Deze folder moet NIET gecommit worden naar de upstream repository. Committen naar de eigen fork van die repository wordt daarmee ook afgeraden omdat het dan erg lastig wordt de folder niet naar de upstream repository te committen.
```

Comment:
Ik weet niet of deze uitleg duidelijk is. De echte reden om de node_modules map + inhoud niet te committen is omdat het geen assets zijn van het project. Bestanden die geen assets vormen van het project kun je door git laten negeren door deze in het .gitignore bestand op te nemen.

Reply:
Puur uit nieuwsgierigheid, waarom wordt de folder 'node_modules' eigenlijk lokaal gegenereerd? Is dat om de scripts ook lokaal te kunnen draaien?

Reply:
De node_modules map bevat de packages die horen bij de tooling of libraries die zijn geïnstalleerd met npm install. Zoals de naam suggereert zijn het node (node.js/javascript) modules.